farmdata2_modules/fd2_tabs/fd2_example/ui.html

```diff
@@ -176,11 +213,56 @@
 
         waitingOnLogs: false,
         seedingLogs: [],
+
+        regularExp: null,
+        selectedRegOpt: null,
+        regexInputVal: null,
+        regexMatchVal: false,
+        disabledMap: {
+            "DropdownWithAllComponent": false,
+            "RegexInputComponent": false,
+            "DateSelectionComponent": false,
+            "DateRangeSelectionComponent": false,
+        }
     },
     methods: {
+        checkDisabled(key) {
```

Comment:
This function name does not seem to describe the function's purpose.  It seems to return the label for the disable/enable button.  Is there a better name?  Alternatively, I think it would be more "Vue-like" to do this with 2 buttons and a v-if.